CHANGELOG.md

```diff
@@ -23,10 +23,15 @@ planned for 2025-07-01
 - [workflow] Run linter und spellcheck with LTS node version (#3767)
 - [workflow] Split "Run test" step into two steps for more clarity (#3767)
 
+### Fixed
+
+- [calendar] fix fullday event rrule until with timezone offset (#3781)
+ 
 ### Updated
 
 - [core] Update dependencies
 
+
```

Comment:
I think prettier don't want this additional line.

Reply:
it was in the source changelog

 i added above it

prettier on system passes
prettier in test run fails

Reply:
from develop branch

![Screenshot at 2025-05-09 17-09-52](https://github.com/user-attachments/assets/26efe42e-e12f-40d4-b246-6f7fd6bcd152)

Reply:
`develop` has only 1 blank line, you have 2 as already stated above